Sources/Plasma/PubUtilLib/plAudio/plWin32StreamingSound.cpp

```diff
@@ -56,6 +56,9 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "pnMessage/plSoundMsg.h"
 #include "pnMessage/plEventCallbackMsg.h"
 #include "plStatusLog/plStatusLog.h"
+#include "plMessage/plSubtitleMsg.h"
+
+#include <regex>
```

Comment:
The other changes in this file are whitespace only...  Are these extra includes really needed?